pkg/vm/vm_ops_maths_test.go

```diff
@@ -117,3 +117,57 @@ func TestSubOp(t *testing.T) {
 	assert.Equal(t, int64(-10), item.Value().Int64())
 
 }
+
+func TestAbsOp(t *testing.T) {
+
+	v := VM{}
+
+	a, err := stack.NewInt(big.NewInt(-20))
+	if err != nil {
+		t.Fail()
+	}
+
+	ctx := stack.NewContext([]byte{})
+	ctx.Estack.Push(a)
+
+	v.executeOp(stack.ABS, ctx)
+
+	// Stack should have one item
+	assert.Equal(t, 1, ctx.Estack.Len())
+	assert.Equal(t, -1, ctx.Astack.Len())
```

Comment:
Let's remove this assertion: `assert.Equal(t, -1, ctx.Astack.Len())` It's not consistent with the other tests, we can refactor a function out at a later stage to test the ctx object

Reply:
:+1: